Alignment/HIPAlignmentAlgorithm/src/HIPAlignmentAlgorithm.cc

```diff
@@ -1268,7 +1268,7 @@ bool HIPAlignmentAlgorithm::calcParameters(Alignable* ali , int setDet, double s
  
   if ( (setDet==0) && (nhit < theMinimumNumberOfHits)) {
     par->setValid(false);
-   edm::LogWarning("Alignment") << "nhit="<<nhit<<" too small,skip!" ;
+    //edm::LogInfo("HIPAlignmentAlgorithm") << "nhit=" << nhit << " too small,skip!";
```

Comment:
The log file becomes huge if the if statement passes, and it is not really an instructive check anyway.